po/modules/options/options.pot

```diff
@@ -2002,6 +2002,10 @@ msgstr ""
 msgid "Require Ctrl to be held down to copy text"
 msgstr ""
 
+#: src/modules/options/OptionsWidget_interfaceFeatures.cpp:55
```

Comment:
Theres no point adding the po changes, please drop

See https://github.com/kvirc/KVIrc/wiki/Updating-KVIrc's-translations